code/modules/projectiles/gun.dm

```diff
@@ -791,26 +791,12 @@
 /obj/item/gun/proc/getinaccuracy(mob/living/user, bonus_spread, stamloss)
 	if(inaccuracy_modifier == 0)
 		return bonus_spread
-	var/base_inaccuracy = weapon_weight * 25 * inaccuracy_modifier
```

Comment:
This is stock Citadel, why is it being removed? This wasn't mentioned in the PR and isn't part of the armor system or penetration system.